scripts/operations/configuration/python_lib/__init__.py

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
Nothing wrong here, but I would point out that you could just leave this file empty unless you really want to document the module in the `__init__.py` doc string or something.

As of https://github.com/Cray-HPE/license-checker/commit/161ab1dbfba509d4dd89826f70ca21bf9fd7e0f8 the license checker passes empty files.

Reply:
Oh, cool -- I just copied this from another file in the repo. No need to put it there if it isn't needed